lib/libzfs/libzfs_pool.c

```diff
@@ -4169,13 +4168,59 @@ zpool_get_errlog(zpool_handle_t *zhp, nvlist_t **nverrlistp)
 	/*
 	 * Fill in the nverrlistp with nvlist's of dataset and object numbers.
 	 */
+	nvlist_t *nv;
+	uint64_t *block_ids;
+	int64_t *indrt_levels;
+	unsigned int error_count = 0;
 	for (i = 0; i < count; i++) {
-		nvlist_t *nv;
 
 		/* ignoring zb_blkid and zb_level for now */
 		if (i > 0 && zb[i-1].zb_objset == zb[i].zb_objset &&
-		    zb[i-1].zb_object == zb[i].zb_object)
+		    zb[i-1].zb_object == zb[i].zb_object) {
```

Comment:
While you're changing this code, can you fix the `i-1`s in this if statement to be `i - 1`, which is what the style guideline says they should be?